configs/stash-origin/config.d/50-stash-origin-authz.cfg

```diff
@@ -26,7 +26,7 @@ if named stash-origin-auth
         -key:/etc/grid-security/xrd/xrdkey.pem \
         -crl:1 \
         -authzfun:libXrdLcmaps.so \
-        -authzfunparms:--lcmapscfg=/etc/xrootd/lcmaps.cfg,--no-authz \
+        -authzfunparms:lcmapscfg=/etc/xrootd/lcmaps.cfg,no-authz \
```

Comment:
I agree with the other changes. But I have no experience with the origin. I would expect you want the authz there.

Reply:
We use the authfile to access control. For example, https://topology.opensciencegrid.org/stashcache/origin-authfile?fqdn=hcc-stashcache-origin.unl.edu